test/flutter_checks.dart

```diff
@@ -169,3 +169,8 @@ extension TableChecks on Subject<Table> {
 extension IconButtonChecks on Subject<IconButton> {
   Subject<bool?> get isSelected => has((x) => x.isSelected, 'isSelected');
 }
+
+extension OffsetChecks on Subject<Offset> {
```

Comment:
nit: keep the file organized more or less logically, rather than in the order we needed things; so put this near the top because it's from `dart:ui`, i.e. the engine; specifically just above Rect, because it's closely related to Rect but simpler